src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--29 calendar--start-day--thu">
+      <section class="calendar__day calendar__day--1"></section>
```

Comment:
```suggestion
    <section class="calendar calendar--month-length--29 calendar--start-day--thu">
      <div class="calendar__day calendar__day--1"></div>
```
Не совсем понятен выбор section для дней календаря, этот тег используется для отдельных семантических блоков, я бы сделал так

Reply:
Слушно! дякую)